### PR TITLE
Revert "Add metrics for sync result"

### PR DESCRIPTION
**Pull request by sawsa307**
Revert #1911

### CODE REVIEW

File: pkg/neg/syncers/transaction.go [hunk 1]
```diff
@@ -381,35 +376,27 @@ func (s *transactionSyncer) CheckEndpointInfo(eds []negtypes.EndpointsData, endp
...
+		return false, negtypes.ResultEPSEndpointCountZero
 	}
 
 	if countFromEndpointData != countFromPodMap {
 		s.logger.Info("Detected error when comparing endpoint counts", "endpointData", eds, "endpointPodMap", endpointPodMap, "dupCount", dupCount)
-		return false, negtypes.NewNegSyncResult(negtypes.ErrEPCountsDiffer, negtypes.ResultEPCountsDiffer)
+		return false, negtypes.ResultEPCountsDiffer
 	}
-	return true, negtypes.NewNegSyncResult(nil, negtypes.ResultSuccess)
+	return true, ""
```
Comment (swetharepakula) [inline pkg/neg/syncers/transaction.go:386]: do you want to keep this as negtypes.ResultSuccess to keep this as a revert?

File: pkg/neg/syncers/transaction.go [hunk 2]
```diff
@@ -381,35 +376,27 @@ func (s *transactionSyncer) CheckEndpointInfo(eds []negtypes.EndpointsData, endp
...
-	if errors.Is(err, negtypes.ErrNodeNotFound) {
-		return true, negtypes.NewNegSyncResult(err, negtypes.ResultNodeNotFound)
-	}
-	if err != nil {
-		return true, negtypes.NewNegSyncResult(nil, negtypes.ResultOtherError)
-	} else {
-		return true, negtypes.NewNegSyncResult(nil, negtypes.ResultSuccess)
+		return false, negtypes.ResultEPMissingZone
 	}
+	return true, ""
```
Comment (swetharepakula) [inline pkg/neg/syncers/transaction.go:399]: same comment as above

**Review by swetharepakula — COMMENTED** [review]
a few nits to stay true to the revert.

File: pkg/neg/syncers/transaction.go (hunk 2 above)
Comment (sawsa307) [reply pkg/neg/syncers/transaction.go]: Updated.

File: pkg/neg/syncers/transaction.go (hunk 1 above)
Comment (sawsa307) [reply pkg/neg/syncers/transaction.go]: Updated.

**Review by swetharepakula — COMMENTED** [review]
/lgtm
/approve
/ok-to-test

**Pull request merged**